scripts/create_markets_from_sheet_omen.py

```
@@ -0,0 +1,100 @@
+from datetime import datetime
```

Ruff

`datetime.datetime` imported but unused

Remove unused import: `datetime.datetime`

(F401)